docs/projects/cn0540/index.rst

```diff
@@ -0,0 +1,358 @@
+.. _cn0540:
+
+CN0540 HDL project
+================================================================================
+
+Overview
+--------------------------------------------------------------------------------
+
+The HDL reference design for the :adi:`CN0540` provides a high resolution, 
+wide-bandwidth, high dynamic range, Integrated Electronics Piezoelectric (IEPE)
+compatible interface data acquisition system (DAQ) for interfacing with
+Integrated Circuit Piezo (ICP)/IEPE piezo vibration sensors. Most solutions 
+which interface with piezo sensors in the market are AC coupled, lacking DC and
+sub-hertz measurement capability. This reference design is a DC coupled solution
+in which DC and sub-hertz precision are achieved. By looking at the complete 
+data set from the vibration sensor in the frequency domain (DC - 50 kHz), the
+type and source of a machine fault can be better predicted using the; position,
+amplitude and number of harmonics found in the FFT spectrum.
+
+The data acquisition board incorporates a precision 24-bit, 1024kSPS Sigma-delta
+ADC :adi:`AD7768-1` and a 16-bit voltage output DAC :adi:`LTC2606`. Used as the
+ADC driver is a high linearity FDA :adi:`ADA4945-1` and a 200mA programmable 
+2-terminal current source :adi:`LT3092`. Analog input protection is provided by
+the switch :adi:`ADG5421F`.
+
+The design have a SPI Engine instance to control and acquire data from the 
+:adi:`AD7768-1` 24-bit precisions ADC, providing support to capture continuous samples
```

Comment:
Please wrap the text to fit within 80 characters/line. You may use as a delimiter the --- symbols above.

Reply:
Solved in V3 commit.